larq/constraints.py

```diff
@@ -34,13 +36,13 @@ class WeightClip(tf.keras.constraints.Constraint):
     clip_value: The value to clip incoming weights.
     """
 
-    def __init__(self, clip_value=1):
+    def __init__(self, clip_value: float = 1):
         self.clip_value = clip_value
 
-    def __call__(self, x):
+    def __call__(self, x: tf.Tensor) -> tf.Tensor:
         return tf.clip_by_value(x, -self.clip_value, self.clip_value)
 
-    def get_config(self):
+    def get_config(self) -> Dict:
```

Comment:
General Python typehints question: Is using the builtin `dict` equivalent to `typings.Dict` in this case?
```suggestion
    def get_config(self) -> dict:
```

Reply:
Not 100% sure about the difference, but now I look into it apparently it's better to use `typing.Mapping` for such things, so I will edit.